init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+:CheckOS
+IF "%PROCESSOR_ARCHITECTURE%"=="x86" (set bit=x86) else (set bit=x64)
```

Comment:
Actually, after talking around I don't think we need the architecture passed around at all. When we pull down the binaries, we should just check what the current running architecture is and pull that one down. You can switch between 64 bit and 32 bit by having the right command prompt